include/arch/arm/aarch64/scripts/linker.ld

```diff
@@ -0,0 +1,496 @@
+
+	KEEP(*(IRQ_VECTOR_TABLE))
+
+	KEEP(*(.vectors))
+
+	KEEP(*(.openocd_dbg))
+	KEEP(*(".openocd_dbg.*"))
+
+	/* Kinetis has to write 16 bytes at 0x400 */
+	SKIP_TO_KINETIS_FLASH_CONFIG
+	KEEP(*(.kinetis_flash_config))
+	KEEP(*(".kinetis_flash_config.*"))
+
+	_vector_end = .;
+	} GROUP_LINK_IN(ROMABLE_REGION)
+
+#ifdef CONFIG_CODE_DATA_RELOCATION
+
+#include <linker_relocate.ld>
+
+#endif /* CONFIG_CODE_DATA_RELOCATION */
+
+    SECTION_PROLOGUE(_TEXT_SECTION_NAME_2,,)
+	{
+	_image_text_start = .;
+	*(.text)
+	*(".text.*")
+	*(.gnu.linkonce.t.*)
+
+	/*
+	 * These are here according to 'arm-zephyr-elf-ld --verbose',
+	 * after .gnu.linkonce.t.*
+	 */
+	*(.glue_7t) *(.glue_7) *(.vfp11_veneer) *(.v4_bx)
+
+#include <linker/priv_stacks-text.ld>
+#include <linker/kobject-text.ld>
+
+	} GROUP_LINK_IN(ROMABLE_REGION)
+
+	_image_text_end = .;
+
+#if defined (CONFIG_CPLUSPLUS)
+	SECTION_PROLOGUE(.ARM.extab,,)
+	{
+	/*
+	 * .ARM.extab section containing exception unwinding information.
+	 */
+	*(.ARM.extab* .gnu.linkonce.armextab.*)
+	} GROUP_LINK_IN(ROMABLE_REGION)
+#endif
+
+	SECTION_PROLOGUE(.ARM.exidx,,)
+	{
+	/*
+	 * This section, related to stack and exception unwinding, is placed
+	 * explicitly to prevent it from being shared between multiple regions.
+	 * It  must be defined for gcc to support 64-bit math and avoid
+	 * section overlap.
+	 */
+	__exidx_start = .;
+#if defined (__GCC_LINKER_CMD__)
+	*(.ARM.exidx* gnu.linkonce.armexidx.*)
+#endif
+	__exidx_end = .;
+	} GROUP_LINK_IN(ROMABLE_REGION)
+
+	_image_rodata_start = .;
+
+#include <linker/common-rom.ld>
+
+    SECTION_PROLOGUE(_RODATA_SECTION_NAME,,)
+	{
+	*(.rodata)
+	*(".rodata.*")
+	*(.gnu.linkonce.r.*)
+
+/* Located in generated directory. This file is populated by the
+ * zephyr_linker_sources() Cmake function.
+ */
+#include <snippets-rodata.ld>
+#ifdef CONFIG_SOC_RODATA_LD
+#include <soc-rodata.ld>
+#endif
+
+#ifdef CONFIG_CUSTOM_RODATA_LD
+/* Located in project source directory */
+#include <custom-rodata.ld>
+#endif
+
+#include <linker/priv_stacks-rom.ld>
+#include <linker/kobject-rom.ld>
+
+	/*
+	 * For XIP images, in order to avoid the situation when __data_rom_start
+	 * is 32-bit aligned, but the actual data is placed right after rodata
+	 * section, which may not end exactly at 32-bit border, pad rodata
+	 * section, so __data_rom_start points at data and it is 32-bit aligned.
+	 *
+	 * On non-XIP images this may enlarge image size up to 3 bytes. This
+	 * generally is not an issue, since modern ROM and FLASH memory is
+	 * usually 4k aligned.
+	 */
+	. = ALIGN(4);
+	} GROUP_LINK_IN(ROMABLE_REGION)
+
+#include <linker/cplusplus-rom.ld>
+
+	_image_rodata_end = .;
+	MPU_ALIGN(_image_rodata_end -_image_rom_start);
+	_image_rom_end = .;
+
+    GROUP_END(ROMABLE_REGION)
+
+/* Some TI SoCs have a special configuration footer, at the end of flash. */
+#ifdef CONFIG_TI_CCFG_PRESENT
+    SECTION_PROLOGUE(.ti_ccfg,,)
+    {
+    KEEP(*(TI_CCFG))
+    } > FLASH_CCFG
+#endif
```

Comment:
This file has some stale entries . Or are these intentional ?

Reply:
Looks like it was copied from `include/arch/arm/cortex_m/scripts/linker.ld`; should be removed.

Reply:
That's correct. I'll clean it up for the next revision.